Xamarin.Forms.Platform.Android/ResourceManager.cs

```diff
@@ -61,23 +61,23 @@ public static int GetResourceByName(string name)
 
 		public static void Init(Assembly masterAssembly)
 		{
-			DrawableClass = masterAssembly.GetTypes().FirstOrDefault(x => x.Name == "Drawable");
-			ResourceClass = masterAssembly.GetTypes().FirstOrDefault(x => x.Name == "Id");
+			DrawableClass = masterAssembly.GetTypes().FirstOrDefault(x => x.Name == "Drawable" || x.Name == "Resource_Drawable");
+			ResourceClass = masterAssembly.GetTypes().FirstOrDefault(x => x.Name == "Id" || x.Name == "Resource_Id");
 		}
 
 		internal static int IdFromTitle(string title, Type type)
 		{
 			string name = Path.GetFileNameWithoutExtension(title);
 			int id = GetId(type, name);
-			return id; // Resources.System.GetDrawable (Resource.Drawable.dashboard);
+			return id;
 		}
 
 		static int GetId(Type type, string propertyName)
 		{
-			FieldInfo[] props = type.GetFields();
-			FieldInfo prop = props.Select(p => p).FirstOrDefault(p => p.Name == propertyName);
-			if (prop != null)
-				return (int)prop.GetValue(type);
+		    object value = type.GetFields().FirstOrDefault(p => p.Name == propertyName)?.GetValue(type)
+				?? type.GetProperties().FirstOrDefault(p => p.Name == propertyName)?.GetValue(type);
```

Comment:
Weird -- I changed the Visual Studio option, but it seems Visual Studio loves its spaces.

Now that I think of it, `memberName` is probably a better argument name than `propertyName`.

Also, I'll be honest -- I wasn't able to get the solution running on my environment, so this request is a best guess rather than fully tested code. I believe it will solve my immediate issue though.